dev-packages/e2e-tests/test-applications/nuxt-3-dynamic-import/tests/tracing.test.ts

```diff
@@ -44,8 +46,22 @@ test.describe('distributed tracing', () => {
       },
     });
 
+    const baggage = (data.headers.baggage || null).split(',');
+
     // connected trace
+    expect(clientTxnEvent.contexts?.trace?.trace_id).not.toBeUndefined();
+    expect(clientTxnEvent.contexts?.trace?.parent_span_id).not.toBeUndefined();
+    expect(baggage).not.toBeNull();
```

Comment:
```suggestion
    expect(baggage).toBeDefined();
```

is a bit easier to read IMHO :)

Reply:
actually also for the tests above 😅